### PR TITLE
Deprecate io/ioutil with go 1.16 and use os and io methods.

### PR DESCRIPTION
**Pull request by provsalt**
As described on https://golang.org/pkg/io/ioutil , ioutil is being deprecated in go 1.16 the same functionality is now provided by package io or package os, and those implementations should be preferred in new code, this PR is aimed to remove io/ioutil

### CODE REVIEW

**Comment by Sandertv** [comment]
Going to put this on hold as this isn't new code and it'd be a waste to force people to update their golang to 1.16 in order to use gophertunnel for no real reason.

**Comment by Sandertv** [comment]
Once generics are released we probably will force the latest version, so this'll probably be merged at that point.

**Review by saadii786 — APPROVED** [review]
Can anyone please guide me how to apply for these settings and how to uese git hub for my mobile infinix note 7 how can i apply and use these settings and souces for my mobile please guide me step by step . NEED HELP

**Comment by provsalt** [comment]
> 
> 
> Can anyone please guide me how to apply for these settings and how to uese git hub for my mobile infinix note 7 how can i apply and use these settings and souces for my mobile please guide me step by step . NEED HELP

I'm sorry. what?

**Comment by DevGod6969** [comment]
> > Can anyone please guide me how to apply for these settings and how to uese git hub for my mobile infinix note 7 how can i apply and use these settings and souces for my mobile please guide me step by step . NEED HELP
> 
> I'm sorry. what?

I think he want to build the apk for the toolbox source code and use it

**Comment by T14Raptor** [comment]
> Can anyone please guide me how to apply for these settings and how to uese git hub for my mobile infinix note 7 how can i apply and use these settings and souces for my mobile please guide me step by step . NEED HELP

This shouldn't be discussed here, please join https://discord.com/invite/U4kFWHhTNR for support, thanks!

**Comment by JustTalDevelops** [comment]
We should be able to merge this in when the v1.18.30 PR is merged, since that PR updates to Go 1.18 too.

**Pull request closed without merging**
